README.md

```diff
@@ -213,3 +213,30 @@ By default the job succeeds without checking anything. If you want to run a secu
 - New repository secret
   - Name: SNYK_TOKEN
   - Secret: Auth token you copied from the Snyk app
+
+# Azure
+
+This repos Docker image: https://hub.docker.com/repository/docker/8kristy/todo-app
+
+## Prerequisites
+- Docker
+- Docker hub account
+- Azure account and resource group
+- Azure CLI
+
+## Deployment
+- Login to Docker using `docker login`
+- Login to Azure using `az login`
+- If you're on Windows, run 
```

Comment:
(Not important)
Worth noting that [Microsoft do actually offer PowerShell for Linux](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-linux?view=powershell-7.4) nowadays - I can't say I've tried it out or can promise it works as hoped, but it _could_ be a team's default option regardless of platform!

(Also, interesting that you're the first learner I can ever remember to default to it! 😄 )